autogen/agentchat/groupchat.py

```diff
@@ -66,11 +66,14 @@ def next_agent(self, agent: Agent, agents: List[Agent]) -> Agent:
 
     def select_speaker_msg(self, agents: List[Agent]):
```

Comment:
Can we keep both versions of select_speaker_msg available for a while? For one, it will be useful for anyone who wants to do comparison and report results. For two, it'll be useful to demonstrate the customizability of this part.
BTW, it may be worth considering writing a blogpost about the changes in this PR and your findings.

Reply:
@sonichi Agree with the blogpost ideas. 

As for the prompt -- a good chunk of the gain comes from the new prompt. But longer-term, prompt customizability is probably what we want. Let me think about how we can maybe parameterize it, or make it quick and easy to subclass.

Reply:
Done.